src/State.js

```diff
@@ -30,20 +30,27 @@ function getStateFromScenes(route, scenes, props) {
   return result;
 }
 
-export function getInitialState(route:{string: any}, scenes:{string:any}, position = 0, props = {}) {
+export function getInitialState(route: {string: any}, scenes:{string:any}, position = 0,
```

Comment:
If this is hitting the line-length rule, let's just break the arguments to separate lines:

```js
export function getInitialState(
  route: {string: any}, 
  scenes:{string:any}, 
  position = 0, 
  props = {}
) {
  // ...
}